fe/src/main/java/org/apache/doris/alter/SchemaChangeHandler.java

```diff
@@ -1405,6 +1420,88 @@ private void sendClearAlterTask(Database db, OlapTable olapTable) {
         LOG.info("send clear alter task for table {}, number: {}", olapTable.getName(), batchTask.getTaskNum());
     }
 
+    public void sendUpdateTabletMetaInfoTask(Database db, OlapTable olapTable, boolean isInMemory) throws DdlException {
+        List<Partition> partitions = Lists.newArrayList();
+        db.readLock();
+        try {
+            partitions.addAll(olapTable.getPartitions());
+        } finally {
+            db.readUnlock();
+        }
+        for(Partition partition: partitions) {
+            sendUpdateTabletMetaTaskForPartition(db, olapTable, partition, isInMemory);
+        }
+    }
+
+    public void sendUpdateTabletMetaTaskForPartition(Database db,
```

Comment:
`sendUpdateTabletMetaTaskForPartition()`

what if this method is partial success? And some of the tablet's meta on BE changed to `in memory` and some are not?

Reply:
If partial success, we will return error to user, and the error msg warn user to retry.
Because this operation is idempotent，So retry is OK.

If we want to avoid partial success, we need to support tablet meta rollback operation. I think which is complex and unnecessary.

Reply:
I am more worried that if the user does not retry, there will be an inconsistent state, that is, some tablet properties on BE are `in memory`, and FE is `not in memory`. There is no mechanism to find it in this case.

We need a correction mechanism to handle this. For example, when reporting to FE, compare with this attribute of FE.

Reply:
OK.